packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-overview.tsx

```diff
@@ -263,9 +263,7 @@ const PullRequestOverview: React.FC<PullRequestOverviewProps> = ({
                                 contentClassName="border-transparent"
                                 content={
                                   <div className="flex py-1">
-                                    <Text size={2} color="primary">
-                                      {commentItem.payload?.payload?.text as string}
-                                    </Text>
+                                    <MarkdownViewer source={commentItem.payload?.payload?.text || ''} />
                                   </div>
                                 }
                                 key={`${commentItem.id}-${commentItem.author}-${idx}`}
```

Comment:
Same here, can avoid index and use only if needed ?

Reply:
done